### PR TITLE
Flattened and annotated skynode v2 p2 Device Tree mapped to P2 Schematic 

### PR DESCRIPTION
**Pull request by davids5**
This is a Work I Progress. 

Items in Red in the [Skynode V2 Pinchart (P1,P2)](https://docs.google.com/spreadsheets/d/1xSTWXg_Jj0kgKaqtW_USwofS817e-jG5XfAv9ESZ2Bo/edit#gid=1245173909) require discussion on initialization and use.

This DT also need to be ported int u-boot to set intial conditions.

### CODE REVIEW

File: arch/arm64/boot/dts/freescale/skynodev2_p2.dts [hunk 1]
```diff
@@ -717,16 +717,6 @@
 	status = "okay";
 };
 
-&rpmsg{
```
Comment (davids5) [inline arch/arm64/boot/dts/freescale/skynodev2_p2.dts]: @ostaebler - what is this and why would it bu used?

Comment (davids5) [reply arch/arm64/boot/dts/freescale/skynodev2_p2.dts]: @ostaebler - there are 2 items left in Pin chart we should discuss when you are back.

**Comment by davids5** [comment]
@caleridas - please test locally A/B compare boot message. If it is == Please  merge and wire it in.

File: arch/arm64/boot/dts/freescale/skynodev2_p2.dts (hunk 1 above)
Comment (ostaebler) [reply arch/arm64/boot/dts/freescale/skynodev2_p2.dts]: RPMSG is a _service_ to communicate with a co-processor (e.g cortex m4) which we do not use.

**Comment by ostaebler** [comment]
> this PR is superseded? can it be dropped?

Yes, I would say so. #19 squashed all commits in this PR.

**Pull request closed without merging**